### PR TITLE
return config and project area data alongside scenarios

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

**Comment by elsiel23** [comment]
@lattemilk FYI

**Review by strangeties — CHANGES_REQUESTED** [review]
lgtm!

Can you please update tests or add a new test to show that project areas are also returned?

**Comment by elsiel23** [comment]
Updated tests

**Review by strangeties — APPROVED** [review]

File: src/planscape/plan/tests.py [hunk 1]
```diff
@@ -1311,17 +1311,20 @@ def test_get_scenario_ok(self):
             {'id': self.scenario.pk},
             content_type="application/json")
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.json()['owner'], self.user.pk)
-        self.assertEqual(response.json()['project'], self.project.pk)
-        self.assertEqual(response.json()['plan'], self.plan.pk)
-        self.assertEqual(response.json()['notes'], 'my note')
-        self.assertEqual(response.json()['priorities'], {
+        scenario = response.json()
+        self.assertEqual(scenario['owner'], self.user.pk)
```
Comment (lattemilk) [inline src/planscape/plan/tests.py:1315]: is project config info also being returned in scenario response? (max_road_distance, max budget, etc)

Comment (elsiel23) [reply src/planscape/plan/tests.py:1315]: @lattemilk it is, i can make that evident in the tests too

**Pull request merged**